applications/constants.py

```diff
@@ -0,0 +1,36 @@
+"""Bootcamp application constants"""
+
+from enum import Enum
+
+
+class SubmissionTypes(Enum):
+    """Enum of possible bootcamp application submission types"""
+    VIDEO_INTERVIEW = "video_interview"
+    QUIZ = "quiz"
+
+
+VALID_SUBMISSION_TYPE_CHOICES = zip(
```

Comment:
Can you add a `list(...)` around this and the other uses in this file? The `zip` will be exhausted after the first use.

Reply:
Great catch